config.sample.js

```diff
@@ -30,16 +30,14 @@ module.exports = {
          /**
           * For updating commit status via the github API
           *
-          * Passed straight through to github.authenticate()
-          * from the `github` npm module:
-          * https://github.com/ajaxorg/node-github
-          * So it accepts whatever that function accpets.
+          * Passed straight through to Octokit
+          * from the `@octokit/rest` npm module,
+          * so it accepts whatever that function accepts.
           *
           * Get an oauth token by going to https://github.com/settings/tokens
-          * and createa token with only repo:status permission
+          * and create a token with only repo:status permission.
           */
          auth: {
-            type: 'oauth', // or 'basic'
             token: 'abcdefghijklmnopqrs...'
          }
```

Comment:
Maybe we should just have 

```
token: 'asdfasdfasdf'
```
 
instead of an auth object now that we only support oath? Not a big deal tbh.

Reply:
I think the only downside is that we'd have to update all our configs

Reply:
true, do we have to update them now that we dropped the `type`? I think it should be fine.